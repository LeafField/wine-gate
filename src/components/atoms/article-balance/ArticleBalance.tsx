import React, { FC } from "react";

type Props = {
  title: string;
  balance: number;
};

const ArticleBalance: FC<Props> = ({ balance, title }) => {
  return (
    <p className="text-main-text">
      {title}:{balance}
    </p>
  );
};

export default ArticleBalance;
