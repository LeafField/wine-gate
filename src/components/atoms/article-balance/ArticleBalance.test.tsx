import { screen, render } from "@testing-library/react";
import ArticleBalance from "./ArticleBalance";

describe("ArticleBalanceの単体テスト", () => {
  test("ArticleBalanceが正しく表示される", () => {
    const { container } = render(<ArticleBalance title="甘味" balance={3} />);
    expect(screen.getByText("甘味:3")).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
