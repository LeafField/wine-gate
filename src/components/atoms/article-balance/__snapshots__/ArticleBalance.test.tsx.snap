// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleBalanceの単体テスト ArticleBalanceが正しく表示される 1`] = `
<div>
  <p
    class="text-main-text"
  >
    甘味
    :
    3
  </p>
</div>
`;
